tests/testthat/helper-functions.R

```diff
@@ -351,18 +351,9 @@ tests_run <- function(dir = ".", filter = NULL) {
 #' @seealso \link{tests_run}
 #' @export
 tests_exit <- function() {
-  res <- stop_binary()
+  stop_binary()
```

Comment:
can you please delete this line? and the stop_binary function? (was related to phantomJS, should have been removed in #126 )